2012-02-08  Rakesh Pandit  <rakesh@fedoraproject.org>

	* locales/ks_IN@devanagari: Fix week day names.

--- a/localedata/locales/ks_IN@devanagari	
+++ a/localedata/locales/ks_IN@devanagari	
@@ -64,11 +64,11 @@ abday       "<U0906><U0925><U094D>";/
             "<U092C><U091F>"
 %
 % Full weekday names (%A)
-day         "<U0906><U0925><U094D><U094D><U0935><U093E><U0930><U094D>";/
+day         "<U0906><U0925><U094D><U0935><U093E><U0930><U094D>";/
             "<U091A><U0928><U094D><U0926><U094D><U0930><U0935><U093E><U0930><U094D>";/
-            "<U092C><U094B><U092E><U094D><U0935><U093E><U0930><U0940><U094D>";/
-            "<U092C><U094B><U0918><U0935><U093E><U0930><U0940><U094D>";/
-            "<U092C><U094D><U0930><U0938><U094D><U094D><U0935><U093E><U0930><U094D>";/
+            "<U092C><U094B><U092E><U094D><U0935><U093E><U0930><U094D>";/
+            "<U092C><U094B><U0918><U0935><U093E><U0930><U094D>";/
+            "<U092C><U094D><U0930><U0938><U094D><U0935><U093E><U0930><U094D>";/
             "<U091C><U0941><U092E><U093E><U0939><U094D>";/
             "<U092C><U091F><U0935><U093E><U0930><U094D>"
 %
